app/src/main/java/org/mozilla/fenix/library/history/HistoryView.kt

```diff
@@ -31,8 +33,25 @@ class HistoryView(
     var mode: HistoryFragmentState.Mode = HistoryFragmentState.Mode.Normal
         private set
 
-    val historyAdapter = HistoryAdapter(interactor)
+    val historyAdapter = HistoryAdapter(interactor).apply {
+        addLoadStateListener {
+            // first call will always have itemCount == 0, but we want to keep adapterItemCount
+            // as null until we can distinguish an empty list from populated, so updateEmptyState()
+            // could work correctly
```

Comment:
Can we fix up this comment by capitalizing and adding a period to the end.

Reply:
fixed